### PR TITLE
Hotfix: adds OCR to checked footer links

### PR DESCRIPTION
**Pull request by anselmbradford**
Footer tests were looking for Office of Civil rights.

## Additions

- Adds OCR to shared footer test.

## Changes

- Linting indent errors.

## Review

- @KimberlyMunoz 
- @sebworks

### CODE REVIEW

File: test/browser_tests/spec_suites/shared/shared_footer.js [hunk 1]
```diff
@@ -8,6 +8,8 @@ describe( 'The Footer Component', function() {
   var _footerLinkLookup =
   { '/offices/accessibility/':
       'accessibility',
+    '/offices/office-of-civil-rights/':
+      'Office of Civil Rights',
```
Comment (sebworks) [inline test/browser_tests/spec_suites/shared/shared_footer.js]: should be lowercase

Comment (anselmbradford) [inline test/browser_tests/spec_suites/shared/shared_footer.js]: Fixed in https://github.com/cfpb/cfgov-refresh/commit/71920e9c47d19820f9b6a2daa60d56addc28aa1b

**Comment by anselmbradford** [comment]
Fixed linting errors while I was at it.

**Comment by sebworks** [comment]
:+1:

**Pull request merged**